apps/android/app/src/main/java/chat/simplex/app/SimplexApp.kt

```diff
@@ -82,7 +82,11 @@ class SimplexApp: Application(), LifecycleEventObserver {
   }
 
   fun allowToStartServiceAfterAppExit() = with(chatModel.controller) {
-    appPrefs.runServiceInBackground.get() && isIgnoringBatteryOptimizations(chatModel.controller.appContext)
+    appPrefs.notificationsMode.get() == NotificationsMode.SERVICE.name && isIgnoringBatteryOptimizations(chatModel.controller.appContext)
+  }
+
+  private fun allowToStartPeriodically() = with(chatModel.controller) {
+    appPrefs.notificationsMode.get() == NotificationsMode.PERIODIC.name && isIgnoringBatteryOptimizations(chatModel.controller.appContext)
```

Comment:
Did you write that you made it independent of battery optimisations? here it seems dependent...

Reply:
Oh, forgot about that, thanks